src/test/java/com/google/api/generator/gapic/composer/FileDiffInfraDummyTest.java

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.api.generator.gapic.composer;
```

Comment:
Would prefer for this file to be located in an isolated directory, since our golden tests may be used elsewhere too.

Reply:
yes, golden tests can be used outside composer folder. That sounds reasonable, removed the test class to a separate folder `dummy`. Thank you!